torch_xla/csrc/ops/index_ops.cpp

```diff
@@ -58,6 +60,21 @@ std::vector<at::Tensor> ExpandByteTensors(const at::Tensor& self,
   return result;
 }
 
+struct IndexAdjacencyInfo {
+  bool contiguous_non_null;
```

Comment:
Init to avoid garbage in case someone instantiate this with default ctor.

Reply:
Done.